Validation/HGCalValidation/interface/validHit.h

```diff
@@ -0,0 +1,43 @@
+#ifndef _validHit_h_
+#define _validHit_h_
+
+#include "TObject.h"
+#include <iostream>
+#include <vector>
+
+class validHit {
```

Comment:
Class names should start with capital letter
```suggestion
class ValidHit {
```